editor/editor_node.cpp

```diff
@@ -47,6 +47,11 @@
 #include "core/string/print_string.h"
 #include "core/string/translation.h"
 #include "core/version.h"
+#include "editor/editor_data.h"
+#include "editor/editor_folding.h"
+#include "editor/editor_run.h"
+#include "editor/editor_toaster.h"
+#include "editor/scene_tree_dock.h"
```

Comment:
This class seems to have `editor/` includes intentionally separated from the rest further below.